### PR TITLE
Trying enabling container build

### PR DESCRIPTION
**Pull request by dantleech**
Just a test to see if container builds can work.

### CODE REVIEW

**Comment by dantleech** [comment]
This works -- but pull requests from forked repositories do not, I created a travis issue: https://github.com/travis-ci/travis-ci/issues/3199

**Comment by WouterJ** [comment]
If I understand your PR on core docs correctly, this should now work for forked repositories too? Btw, there are lots of false negatives now :(

**Comment by dantleech** [comment]
Yeah I tested it and it seemed to work. For the false positives, I guess it might be an older version of Enchant or something, the false positives look like they should be in the standard dictionary, will investigate.

**Comment by WouterJ** [comment]
btw, your last commit came into the wrong PR?

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+
 python:
     - "2.7"
 
-before_script:
-    - sudo apt-get install enchant
+addons:
+    apt_packages:
+        - enchant
+        - foobar
```
Comment (dbu) [inline .travis.yml]: really?

Comment (dantleech) [inline .travis.yml]: :) was just to test

**Comment by dantleech** [comment]
Closing this for now.

**Comment by dbu** [comment]
@dantleech can we delete this branch?

**Comment by dantleech** [comment]
done

**Pull request closed without merging**